.github/workflows/release.yml

```diff
@@ -60,23 +60,26 @@ jobs:
     environment: release
     runs-on: ubuntu-22.04
     needs: pypi
+    if: github.event_name == 'release'
 
     env:
       FORCE_COLOR: 1
       PY_COLORS: 1
       TOXENV: pkg
 
     steps:
-      - name: Check out src from Git
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0 # needed by setuptools-scm
-          submodules: true
+      - name: Extract version
+        id: extract-version
+        env:
+          TAG_NAME: ${{ github.event.release.tag_name }}
+        run: |
+          echo "version=${TAG_NAME#v}" >> $GITHUB_OUTPUT
```

Comment:
This is all just to transform `vX.Y.Z` to `X.Y.Z` (i.e. strip the `v` prefix), since GitHub Actions workflow syntax does not have a function that does that.